### PR TITLE
add initial testnet alerts and pagerduty setup

### PR DESCRIPTION
**Pull request by 0x0I**
Scaffolding and setup for defining and deploying Prometheus => Alertmanager => Pagerduty alerting flows

- rules and alert-receiver configs are uploaded to and managed by Grafana Cloud [alertmanager services](https://o1testnet.grafana.net/a/grafana-alerting-ui-app/?tab=rules&rulessource=grafanacloud-o1testnet-prom)
- linting and uploading of alerts to Grafana Cloud are executed via the `cortextool`, maintained by Grafana

### CODE REVIEW

**Review by bkase — APPROVED** [review]
Awesome! Can you change the 15m to 1hr?

**Comment by bkase** [comment]
What is the consequences of not putting this into a secrets manager? I think it should be okay for now to land. I know Aneesha was mentioning this may be important though.

**Comment by 0x0I** [comment]
> What is the consequences of not putting this into a secrets manager? I think it should be okay for now to land. I know Aneesha was mentioning this may be important though.

Seems like the primary consequence of not putting the integration key into some sort of secret manager (/leaving it publicly visible) would be opening up the on-call alert triggering flow to spam/spoof attacks (technically anyone with the key can send alerts though need to double-check that).

Afaict - there's not a lot of guidance around it from [Pagerduty](https://support.pagerduty.com/docs/services-and-integrations) or the community though I'm thinking the expectation is to just treat as any other "API" key.

File: terraform/infrastructure/us-east1.tf [hunk 1]
```diff
@@ -29,6 +29,18 @@ locals {
...
     }
+    serverFiles = {
+      "alerting_rules.yml" = local.testnet_alerts
+    }
+    alertmanagerFiles = {
+      "alertmanager.yml" = {
+        receivers = local.pagerduty_receivers
+        route = {
+          group_by = "[testnet]"
+          receiver = "pagerduty-primary"
```
Comment (0x0I) [inline terraform/infrastructure/us-east1.tf:40]: update

**Review by lk86 — APPROVED** [review]

**Comment by 0x0I** [comment]
migrated to *mina* repo: https://github.com/MinaProtocol/mina/pull/7430

**Pull request merged**